.github/pr.yml

```diff
@@ -0,0 +1,28 @@
+name: pr
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  packages: write
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: "remear/reactor-developer-docs"
+
+jobs:
+  docker:
```

Comment:
I'm thinking we could employ a slightly better job name here to convey at least the job's purpose. What do you think about using `build` for now?